src/main/scala/dxWDL/Main.scala

```diff
@@ -333,13 +337,13 @@ object Main extends App {
         )
     }
 
-  private def parseStreamAllFiles(s: String): Boolean = {
+  private def parseStreamFiles(s: String): StreamFiles.StreamFiles = {
     s.toLowerCase match {
-      case "true"  => true
-      case "false" => false
+      case "all"   => StreamFiles.All
+      case "false" => StreamFiles.None
```

Comment:
One more thing: shouldn't it be "none" instead of false. Also, there should be a None (per-file) case for when the flag is not used (default)?

Reply:
good catch